src/app/v2/trade/chart/index.tsx

```diff
@@ -0,0 +1,85 @@
+import { useRef, useEffect } from 'react';
+import { createChart, IChartApi, CandlestickData } from 'lightweight-charts';
+import { tailwindConfig } from '@penumbra-zone/ui/tailwind';
+import { Token } from '@/utils/types/token';
+import { useCandles } from '@/fetchers/candles';
+
+const { colors } = tailwindConfig.theme.extend;
+
+interface ChartProps {
+  height: number;
+}
+
+const asset1: Token = {
+  display: 'Penumbra',
+  symbol: 'um',
+  decimals: 5,
+  inner: '',
+};
+const asset2: Token = {
+  display: 'GM Wagmi',
+  symbol: 'gm',
+  decimals: 5,
+  inner: '',
+};
+
+export function Chart({ height }: ChartProps) {
```

Comment:
suggestion/nitpick (non-blocking for now): move the `chart` directory into the `components`. Even though it is tempting to store all components related to the trade page nearby, I think we should separate it